.github/workflows/daily.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         #this is a painful solution since it doesn't pick up new branches, other option is lotsa shell in one job....
         #to test in canary add in canary here.....
-        target-branch: [main, release-1.1]
+        target-branch: [main, release-1.1, release-1.2]
```

Comment:
nit: neither of these commit messages reflect this change (unless this change is relevant to using the bouncer bot + environment variables, I don't quite understand why it's here).

Commit messages:
- `[gha] use environments for secrets.`
- `[gha] compute environment nameAdding a new branch is needed `

Reply:
true, a missing job in current config in main.